tests/upload_files.rs

```diff
@@ -12,7 +13,16 @@ mod fixtures;
 use crate::fixtures::{server, tmpdir, Error, TestServer};
 
 #[rstest]
-fn uploading_files_works(#[with(&["-u"])] server: TestServer) -> Result<(), Error> {
+#[case("", "")]
+#[case(
```

Comment:
We haven't done any case naming in the project yet but perhaps here it would be appropriate?
```suggestion
#[case::no_hash("", "")]
#[case::sha256_hash(...)
```